collections/Canon_Misc/MS_Canon_Misc_573.xml

```diff
@@ -71,82 +80,191 @@
+                        <msItem>
+                           <locus>(fol. 162va)</locus>
+                           <bibl>Index prohemium</bibl>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 163v-164r)</locus>
+                           <note>blank</note>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 164va-165ra)</locus>
+                           <bibl>Index questionum</bibl>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 165rb-171r)</locus>
+                           <note>blank</note>
+                        </msItem>
                      </msItem>
                   </msContents>
                   <physDesc>
                      <objectDesc form="codex">
-                        <supportDesc material="chart">
-                           <support>paper</support>
+                        <supportDesc material="mixed">
+                           <support>parchment and paper</support>
+                           <extent>171 leaves <dimensions unit="mm" type="leaf">
+                                 <height>296</height>
+                                 <width>205</width>
+                              </dimensions>
+                           </extent>
+                           <foliation>Modern foliation.</foliation>
+                           <collation>1–6<hi rend="superscript">16</hi>, 7<hi rend="superscript">14</hi>, 8–10<hi rend="superscript">16</hi>, 11<hi rend="superscript">16</hi> (wants 3). <catchwords>Catchwords at the foot of the last page of each quire (except for the first two).</catchwords></collation>
                         </supportDesc>
+                        <layoutDesc>
+                           <layout writtenLines="57 62" columns="2">2 columns of 57–62 lines, ruled space <dimensions unit="mm" type="ruled">
+                                 <height>225</height>
+                                 <width>153</width>
+                              </dimensions>
+                           </layout>
+                        </layoutDesc>
                      </objectDesc>
                   </physDesc>
                   <history>
                      <origin>
-                        <origDate calendar="Gregorian" when="1385">1385</origDate>
-                        
-                     </origin>
+                        <origPlace key="place_7006464">Prague</origPlace>
```

Comment:
this should be in the format:
'<origPlace><country key="xxxxxx">Bohemia</country>, <settlement key="place_7006464">Prague</settlement><origPlace>'